SpatialGDK/Documentation/content/helper-scripts.md

```diff
@@ -6,3 +6,4 @@ These scripts are located under `Plugins\UnrealGDK\SpatialGDK\Build\Scripts\` di
 | Helper script | Parameters | Description |
 | --- | --- | --- |
 | `BuildWorker.bat` | `<target> <platform> <configuration> <YourGame>.uproject [--skip-codegen]` | Example: </br> `BuildWorker.bat ExampleGameEditor Win64 Development ExampleGame.uproject` </br></br> Build, cook and zip your Unreal server-workers and client-workers for use with a SpatialOS cloud deployment (uploaded using [`spatial cloud upload`](https://docs.improbable.io/reference/latest/shared/deploy/deploy-cloud)). </br></br>  The following `<target>`s  generate zipped workers: </br> - `<YourGame>` </br> - `<YourGame>Server` </br> - `<YourGame>Editor` </br></br> Any other `<target>` passes all arguments to `Engine\Build\BatchFiles\Build.bat` with no cooking or zipping performed.|
+| `ExternalSchemaCodegen.bat` | `<schema_directory> <output_code_directory>` | Example: </br> `ExternalSchemaCodegen.bat ./spatial/schema/my_external_schema ./Game/Source/ThirdPersonShooter/ExternalSchemaCodegen` </br></br> Run the code generator binary created by the Setup.bat script to generate SpatialOS schema component types and an interface for sending and registering callbacks for receiving SpatialOS component updates.|
```

Comment:
This is a bit of a mind-melt. Have asked a few questions in here.
https://docs.google.com/document/d/1DLrLcDsqrxhLpw1L_GNBGwoQR6kRjpWKr4-reUwsRYU/edit